src/tests/multihost/alltests/test_default_debug_level.py

```diff
@@ -223,8 +250,6 @@ def test_bz785908(multihost, backupsssdconf):
             4. Update sssd domain with ldap_group_object_class, ldap_search_base and
                 ldap_group_object_class
         :steps:
-            1. Runs command getent -s sss group to query the SSSD cache and save the results to a file.
```

Comment:
Just deleting the steps does not look correctly.

Reply:
fixed